<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1141</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="BlankPage"/>
    <widget class="QWidget" name="LoginPage">
     <property name="minimumSize">
      <size>
       <width>821</width>
       <height>0</height>
      </size>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>120</y>
        <width>291</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Login </string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>177</width>
         <height>124</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_username"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_password"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="loginButton">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="AdminPage">
     <widget class="renderArea" name="renderWidget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>1101</width>
        <height>491</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addShapeButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>530</y>
        <width>401</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Shape</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>530</y>
        <width>371</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete Shape</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="UserPage">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>210</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>user</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1140</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLogin"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>renderArea</class>
   <extends>QWidget</extends>
   <header>renderarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
